Clinical-Genomics/scout
# Fix #822 - avoid highlighting matching causatives in unaffected.

This PR adds a functionality or fixes a bug.

**How to test**:
1. With the current master, mark a somewhat common variant causative on stage, say https://scout-stage.scilifelab.se/cust002/F0017223/0501ab658fa97ff11e8522e9d3587ddf
1. view a case where only an unaffected individual carries the variant, say 
https://scout-stage.scilifelab.se/cust002/F0011840/91f873d8f31a7fa5ed37385c777ca1e2
1. note that the variant shows as a "Matching causative" on the case page of the latter case. <img width="280" alt="Screenshot 2019-10-31 at 13 03 40" src="https://user-images.githubusercontent.com/758570/67945332-ef714480-fbde-11e9-9cc9-4aeadfba32a3.png">
1. now update to the PR branch on stage
1. and see that the variant no longer displays as a "Matching causative" on the case where only the unaffected parent carries it.<img width="300" alt="Screenshot 2019-10-31 at 13 19 00" src="https://user-images.githubusercontent.com/758570/67946232-2ba5a480-fbe1-11e9-9fe0-c3ad11468a76.png">
1. Finally, check that a case with true matching causative to other cases is still flagged as such, e.g. https://scout-stage.scilifelab.se/cust002/F0015258

**Expected outcome**:
The functionality should be working

**Review:**
